python/ray/serve/_private/autoscaling_state.py

```diff
@@ -264,10 +251,7 @@ def drop_stale_handle_metrics(self, alive_serve_actor_ids: Set[str]) -> None:
         received an update for too long.
         """
 
-        timeout_s = max(
-            2 * self._config.metrics_interval_s,
-            RAY_SERVE_MIN_HANDLE_METRICS_TIMEOUT_S,
-        )
+        timeout_s = 2.5 * RAY_SERVE_HANDLE_AUTOSCALING_METRIC_PUSH_INTERVAL_S
```

Comment:
can you explain why we are changing this logic? Also the rationale behind `2.5`, why just use `RAY_SERVE_HANDLE_AUTOSCALING_METRIC_PUSH_INTERVAL_S`? A inline code comment here would be nice